geomstats/hypersphere.py

```diff
@@ -1,12 +1,12 @@
 """
-Computations on the n-dimensional sphere
+Class for the n-dimensional hypersphere
```

Comment:
try to avoid mentioning implementation details in the file - you might want to add more stuff in there in the future. Original comment as actually better imho